integrations/drip/drip.go

```diff
@@ -1,31 +1,134 @@
 package drip
 
 import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/jipiboily/forwardlytics/integrations"
 )
 
 // Drip integration
 type Drip struct {
+	api service
+}
+
+type service interface {
+	request(string, string, []byte) error
+}
+
+type dripAPIProduction struct {
+	Url string
+}
+
+type apiSubscriber struct {
+	CustomFields map[string]interface{} `json:"custom_fields"`
+	Email        string                 `json:"email"`
+	UserId       string                 `json:"user_id"`
+}
+
+type apiEvent struct {
+	Action     string                 `json:"action"`
+	Email      string                 `json:"email"`
+	OccurredAt time.Time              `json:"occurred_at"`
+	Properties map[string]interface{} `json:"properties"`
 }
 
 // Identify forwards and identify call to Drip
-func (Drip) Identify(identification integrations.Identification) (err error) {
-	logrus.Errorf("NOT IMPLEMENTED: will send %#v to Drip\n", identification)
+func (d Drip) Identify(identification integrations.Identification) (err error) {
+	s := apiSubscriber{}
+	// Drip needs an email to identify the user
+	if identification.UserTraits["email"] == nil {
+		logrus.Error("Drip: Required field email is not present")
+		return errors.New("Email is required for doing a drip request")
+	} else {
+		s.Email = identification.UserTraits["email"].(string)
+	}
+
+	s.UserId = string(identification.UserID)
+
+	// Add custom attributes
+	s.CustomFields = identification.UserTraits
+	s.CustomFields["forwardlyticsReceivedAt"] = identification.ReceivedAt
+	s.CustomFields["forwardlyticsTimestamp"] = identification.Timestamp
```

Comment:
Can we set the timestamp of the identify call on Drip's end? I know we can for Intercom. It's not huge if can't honestly but it would be better if that's possible.